src/main/webapp/resources/js/pages/projects/project-analyses/components/SingleSampleAnalysisOutputs.jsx

```diff
@@ -0,0 +1,185 @@
+import React from "react";
+import { Button, Input, Table } from "antd";
+
+import { SPACE_MD, SPACE_XS } from "../../../../styles/spacing";
+import { IconDownloadFile } from "../../../../components/icons/Icons";
+
+import { dateColumnFormat } from "../../../../components/ant.design/table-renderers";
+import { setBaseUrl } from "../../../../utilities/url-utilities";
+
+const { Search } = Input;
+
+/**
+ * React component for single sample analysis outputs
+ * @param outputs The single sample analysis outputs
+ * @param isLoading If data is still being retrieved from the server
+ * @param projectId The project identifier
+ * @returns {JSX.Element}
+ * @constructor
+ */
+
+export default function SingleSampleAnalysisOutputs({
+  outputs,
+  isLoading,
+  projectId,
+}) {
+  const [selected, setSelected] = React.useState([]);
+  const [filteredOutputs, setFilteredOutputs] = React.useState(null);
+
+  // Ant Design Table options
+  const PAGE_SIZE_OPTIONS = [5, 10, 25, 50, 100];
+  const DEFAULT_PAGE_SIZE = 10;
+
+  const columns = [
+    {
+      title: i18n("SingleSampleAnalysisOutputs.id"),
+      key: "analysisOutputFileId",
+      dataIndex: "analysisOutputFileId",
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.sampleName"),
+      key: "sampleName",
+      dataIndex: "sampleName",
+      render(sampleName, record) {
+        return (
+          <a
+            href={setBaseUrl(
+              projectId
+                ? `/projects/${projectId}/samples/${record["sampleId"]}`
+                : `/samples/${record["sampleId"]}`
+            )}
+            target="_blank"
+          >
+            {sampleName}
+          </a>
+        );
+      },
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.file"),
+      key: "fileName",
+      dataIndex: "filePath",
+      render(filePath, record) {
+        let splitItems = filePath.split("/");
+        let fileName = splitItems[splitItems.length - 1];
+        return fileName + " (" + record["analysisOutputFileKey"] + ")";
+      },
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.analysisType"),
+      key: "analysisType",
```

Comment:
I think you can set the dataIndex to an array and then you would not need the render method.

`dataIndex: ["workflowDescription", "type"]`

Reply:
I don't think that will work here. The dataIndex is set to analysisType currently which has an object {type: ANALYSIS_TYPE}. That's why the render method was used :(

Reply:
Updated in d5b358d66d7ee753c754ed08653476fbfafabf73. Thanks learned something new